### PR TITLE
chore(sdk): update links for new docs

### PR DESCRIPTION
**Pull request by laxels**
https://www.notion.so/wandbai/New-URL-paths-e4004fae6cbc452f9d46bd9fcf810437

Will not merge/deploy before we switch over to the new docs.

### CODE REVIEW

File: wandb/sdk/lib/wburls.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ def _get_urls(self) -> Dict["URLS", str]:
             cli_launch="https://wandb.me/launch",
             doc_run="https://wandb.me/run",
             doc_require="https://wandb.me/library-require",
-            doc_start_err="https://docs.wandb.ai/library/init#init-start-error",
+            doc_start_err="https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-",
```
Comment (kptkin) [inline wandb/sdk/lib/wburls.py:30]: should we replace this one with a wandb.me link?

File: wandb/sdk/lib/wburls.py [hunk 2]
```diff
@@ -27,7 +27,7 @@ def _get_urls(self) -> Dict["URLS", str]:
...
             doc_artifacts_guide="https://docs.wandb.ai/guides/artifacts",
```
Comment (kptkin) [inline wandb/sdk/lib/wburls.py:31]: maybe this one as well?

**Review by kptkin — APPROVED** [review]
thanks for fixing this! minor suggestion is to replace all explicit links with `wandb.me`. This will allow us to change the links without the need to change the code.

I think ideally (or eventually), we should have only wandb.me links, what do you think?

**Comment by laxels** [comment]
> thanks for fixing this! minor suggestion is to replace all explicit links with `wandb.me`. This will allow us to change the links without the need to change the code.
> 
> I think ideally (or eventually), we should have only wandb.me links, what do you think?

I think that's a great idea for an eventual goal. Noah is aware of it and they may implement it at some point.

File: wandb/sdk/lib/wburls.py (hunk 1 above)
Comment (laxels) [reply wandb/sdk/lib/wburls.py:30]: Good idea. We may eventually do this, but it should be part of a large effort to replace the links in bulk.

File: wandb/sdk/lib/wburls.py (hunk 2 above)
Comment (laxels) [reply wandb/sdk/lib/wburls.py:31]: Good idea. We may eventually do this, but it should be part of a large effort to replace the links in bulk.

**Pull request merged**